File: LeetCode-Programs/How Many Numbers Are Smaller Than the Current Number.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        vector<int> vec;
        for(int i=0;i<nums.size();i++)
        {
            int count=0;
            for(int j=0;j<nums.size();j++)
            {
                if(nums[j]<nums[i])
                {
                    count++;
                }
            }
            vec.push_back(count);
        }
        return vec;
    }
};

File: LeetCode-Programs/How Many Numbers Are Smaller Than the Current Number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        new_list=[]
        for i in nums:
            count=0
            for j in nums:
                if j<i:
                    count=count+1
            new_list.append(count)
        return new_list
